tests/Integration/bootstrap.php

```diff
@@ -52,6 +52,10 @@ function bootstrap_integration_suite( $wp_tests_dir ) {
 	tests_add_filter(
 		'muplugins_loaded',
 		function() {
+			// Overload the license key for testing.
+			\Patchwork\redefine( 'rocket_valid_key', '__return_true' );
```

Comment:
@Tabrisrp @Screenfeed @crystinutzaa This code is important to ensure all files are loading in `inc/main.php` when the `'plugins_loaded'` event fires, ie when the plugin boots up. 

Why do we need to do it here? Why not within the Test Case or as needed within a test itself?

The loading happens _before_ the tests run. It's during the boot up of WordPress and the plugin. So we need to overload the function here to ensure those files load as they would for our customers.